### PR TITLE
HH->bbgg analysis

### PR DESCRIPTION
**Pull request by chernyavskaya**
Working HH->bbgg analysis for 2016 and 2017 eras.

### CODE REVIEW

**Comment by shervin86** [comment]
do we really need to keep track of all these commits? It would be better if you can rebase and squash them.

**Comment by simonepigazzini** [comment]
@shervin86 don't worry, we can squash when merging the PR now. I will squash everything into a single commit.

**Comment by simonepigazzini** [comment]
L 994 in MetaData/python/sample_utils.py breaks all sample management tools. I made a quick fix with Nadya (removed copyProxy option) but I suspect that changes to other file were lost before making the PR

File: MetaData/python/samples_utils.py [hunk 1]
```diff
@@ -984,7 +990,8 @@ def __call__(self):
         self.mn = SamplesManager("$CMSSW_BASE/src/%s/MetaData/data/%s/datasets*.json" % (options.metaDataSrc,options.campaign),
                                  dbs_instance=options.dbs_instance,
                                  force=options.doForce,
-                                 queue=options.queue,maxThreads=options.max_threads,doContinue=options.doContinue)
+                                 queue=options.queue,maxThreads=options.max_threads,doContinue=options.doContinue,
+                                 copyProxy=options.copy_proxy)
```
Comment (simonepigazzini) [inline MetaData/python/samples_utils.py]: @musella @michelif
This breaks all sample management tools. I made a quick fix with Nadya (removed copyProxy option) but I suspect that changes to other file were lost before making the PR

Comment (musella) [reply MetaData/python/samples_utils.py]: The option should be defined. It was lsot ins some manupulation
```
make_option("--no-copy-proxy",dest="copy_proxy",action="store_false",
                            default=True,help="Do not try to copy the grid proxy to the worker nodes."
),
```

**Comment by musella** [comment]
@simonepigazzini 
I think that the changes should be squashed to 3-4 commits
- b regression
- HH analysis
- fixes to job_utils
- fixes to sample_utils

I guess that we can do the homework and make another PR.

**Comment by simonepigazzini** [comment]
merged in: https://github.com/simonepigazzini/flashgg/commit/8e88c73ac652c4c82a1313c4fdb17729c0b6be50

(to be merged later in this repo)

**Pull request closed without merging**